z_test.go

```diff
@@ -3185,3 +3193,227 @@ func TestOpenCloseLob(t *testing.T) {
 		})
 	}
 }
+
+func TestPreFetchQuery(t *testing.T) {
+
+	if os.Getenv("GODROR_TEST_SYSTEM_USERNAME") == "" ||
+		(os.Getenv("GODROR_TEST_SYSTEM_PASSWORD") == "") {
+		t.Skip("Please define GODROR_TEST_SYSTEM_USERNAME and GODROR_TEST_SYSTEM_PASSWORD env variables")
+	}
+	var err error
+	if testSystemDb, err = sql.Open("godror", testSystemConStr); err != nil {
+		panic(errors.Errorf("%s: %+v", testConStr, err))
+	}
+
+	ctx, cancel := context.WithTimeout(testContext("TestPreFetchQuery"), 30*time.Second)
+	defer cancel()
+
+	// Create a table used for Prefetch, ArrayFetch queries
+
+	tbl := "t_employees" + tblSuffix
+	testDb.ExecContext(ctx, "DROP TABLE "+tbl)
+	if _, err := testDb.ExecContext(ctx, "CREATE TABLE "+tbl+" (employee_id NUMBER, last_name VARCHAR2(30))"); err != nil {
```

Comment:
The table doesn't need two columns.  The test would be simpler with just a single column.

Reply:
modified to use single column